Docs.md

```diff
@@ -63,6 +63,92 @@ Or see the [doc website](https://deno.land/typedoc/index.html).
 If you are embedding deno in a Rust program, see
 [the rust docs](https://deno.land/rustdoc/deno/index.html).
 
+## Build Instructions
+
+Follow these steps if you are trying to submit a PR.
```

Comment:
This line seems lost within it's surrounding context.

Reply:
Removed.